src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -946,7 +945,7 @@ private ImmutableArray<Symbol> CreateMembers()
                     // Add a field with default name if the given name is different
                     if (name != defaultName)
                     {
-                        members.Add(new TupleErrorFieldSymbol(this, defaultName, -members.Count - 1, null, _elementTypes[i], diagnosticInfo));
+                        members.Add(new TupleErrorFieldSymbol(this, defaultName, i, null, _elementTypes[i], diagnosticInfo));
```

Comment:
>i [](start = 81, length = 1)

I think this change invalidates a bunch of comments in TupleErrorFieldSymbol.cs.